patches/chrome-browser-importer-profile_writer.cc.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/chrome/browser/importer/profile_writer.cc b/chrome/browser/importer/profile_writer.cc
+index 8196f333154dc5a14ac6c4f9757ac48135d1dafb..a6c193b258cb34889fcb4f8052739506b40b559f 100644
+--- a/chrome/browser/importer/profile_writer.cc
++++ b/chrome/browser/importer/profile_writer.cc
+@@ -106,12 +106,14 @@ void ProfileWriter::AddHistoryPage(const history::URLRows& page,
+     HistoryServiceFactory::GetForProfile(profile_,
+                                          ServiceAccessType::EXPLICIT_ACCESS)
+         ->AddPagesWithDetails(page, visit_source);
++#if !BUILDFLAG(IS_ANDROID)
```

Comment:
the obvious way to get rid of this patch is add into
`brave/chromium_src/chrome/browser/importer/profile_writer.cc`
```c++
#if BUILDFLAG(IS_ANDROID)
namespace first_run {

bool IsChromeFirstRun() {
  return false;
}

}
#endif  // BUILDFLAG(IS_ANDROID)

```